README.Rmd

```diff
@@ -17,49 +17,77 @@ knitr::opts_chunk$set(
 
 <!-- badges: start -->
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
-[![R-CMD-check](https://github.com/The-Strategy-Unit/nhp_inputs/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/The-Strategy-Unit/nhp_inputs/actions/workflows/R-CMD-check.yaml)
+[![check](https://github.com/The-Strategy-Unit/nhp_inputs/actions/workflows/check.yaml/badge.svg)](https://github.com/The-Strategy-Unit/nhp_inputs/actions/workflows/check.yaml)
 <!-- badges: end -->
 
 ## About
 
-An app to input the parameters needed to run scenarios through the New Hospital Programme (NHP) demand model.
+A web app to input the parameters needed to run scenarios through the New Hospital Programme (NHP) demand model.
+Results can then be viewed in [the outputs app](https://github.com/The-Strategy-Unit/nhp_outputs).
```

Comment:
Does this need to be a link to the deployed outputs app rather than the repo - or maybe not if this is not fully public?

Reply:
Thanks both, I agree!